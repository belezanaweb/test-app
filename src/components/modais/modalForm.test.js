import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import modalFormRemember from './modalFormRememberMe';

describe('Component Modal Form Remember', () => {
	test('test Snapshot', () => {
		const tree = renderer.create(<modalFormRemember />).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
