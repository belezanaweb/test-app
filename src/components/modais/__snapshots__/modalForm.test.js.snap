// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Modal Form Remember test Snapshot 1`] = `<modalFormRemember />`;
